docs/source/getting_started/starting.rst

```diff
@@ -11,6 +11,15 @@ Start JupyterLab using:
 
 JupyterLab will open automatically in your browser.
 
+If the location of your notebook files are not in your home directory (more often the case in Windows, or if you have an external drive) you can use the `--notebook-dir` added parameter to start JupyterLab in that directory.  Be very careful though, never run it in your root volume (ex: `c/` in Windows, `/` in Linux) unless you know what you are doing and are prepared for the consequences! 
+
+.. code:: bash
+
+    #Windows Example
+    jupyter lab --notebook-dir E:/
+    #Linux Example
+    jupyter lab --notebook-dir /var/www/html/
```

Comment:
Actually there is a shortcut for that option:

```suggestion
    jupyter lab <path/to/working/directory>
```

Reply:
Also, `--notebook-dir` is deprecated, but `--root-dir` and `--preferred-dir` might be worth mentioning. Combining the two allows to set root to e.g. `/home/user/projects/` but open the browser or `/home/user/projects/current_project/`.

Reply:
`--root-dir' and `--preferred-dir` do not seem to work but `--app_dir` and `--preferred_dir` do. I'm running v3.2.1 .

Reply:
I updated the pending changes to reflect the new parameters